### DISCUSSION TITLE
Does Helm chart argocd- alignment for ArgoCD Security release: v2.6.8 and v2.5.17 already exists somehere?

### DISCUSSION THREAD

**Discussion by good92** [argoproj/argo-helm#2094] [Q&A] [answered]
In https://github.com/argoproj/argo-cd/releases two new release are present, but I did not found them as argocd- Helm chart.
Where can we access them?

ref: https://github.com/argoproj/argo-helm/issues/2093

**Comment by yu-croco** [comment, marked as answer]
Hi @good92 ,
Unfortunately we do not support past version update.
If you want to use v2.6.8 or v2.5.17 in latest Chart version, you need to set image tag on your values.yaml.

**Reply by yu-croco** [reply]
> you need to set image tag on your values.yaml.

It's here.
https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L58
